HW05/hw05.Rmd

```diff
@@ -25,82 +31,277 @@ Each dataset contains the following information for the Top 3 selling scented or
+#  1. Date on the x axis
+#    1. Hint: use `scale_x_date()`
+#    2. Format the date labels as `"%m-%Y"`
+#    3. Set the date breaks to be "6 months"
+#  2. **Average daily rating** of each candle on the y axis
+#  3. A smooth trend line (not linear; `method = "loess"`)
+#  4. A dashed vertical line at January 01, 2020
+#    1. Hint: use `geom_vline()` and set `xintercept = as.numeric(as.Date("2020-01-20"))`
     
-Be sure to make the plots beautiful.
+#Be sure to make the plots beautiful.
 
-  1. Address overplotting of points using transparency or shapes.
-  2. Use an appealing theme and color scheme.
-  3. Set appropriately formatted axis titles and main plot title.
-  4. Place the two plots side by side.
+#  1. Address overplotting of points using transparency or shapes.
+#  2. Use an appealing theme and color scheme.
+#  3. Set appropriately formatted axis titles and main plot title.
+#  4. Place the two plots side by side.
   
-Note: The raw data are **individual reviews** so you will need to compute summary results to get average daily ratings for the desired date range.
+#Note: The raw data are **individual reviews** so you will need to compute summary results to get average daily ratings for the desired date range.
 
-Interpret these graphs.
-What do they show about scented candles over the time period?
+#Interpret these graphs. What do they show about scented candles over the time period?
+
+```
 
-```{r}
+
+```{r Part 1 Code, message=FALSE, results='hide', warning = FALSE}
+
+#Uploading scented data
+dat.scented <- read_csv(here::here("Scented_all.csv"),
+                        col_types=cols(
+                          CandleID = col_character(),
+                          Date = col_character(),
+                          Rating = col_double(),
+                          Review = col_character()
+                          )
+                        )
+
+#Adjust scented date data to correct format                
+dat.scented$Date = as.Date(dat.scented$Date, "%d-%b-%y")
+
+#Filter scented data to dates after 2016 and ID below 4
+cleaned.scented <- dat.scented %>% 
+  filter(Date >= as.Date("2017-01-01") & CandleID <= 3)
+
+#Summarize ratings by ID and Date
+mean.scented <- cleaned.scented %>%
+  group_by (CandleID, 
+            Date
+            ) %>% 
+  summarize(across(c(Rating),
+                     list(Mu = ~ mean(.x,
+                                      na.rm = TRUE
+                                     ),
+                         Sigma = ~ sd(.x,
+                                      na.rm = TRUE
+                                     ),
+                         n = ~ n()
+                         )
+                  )
+            )
+
+#Plot
+plot.scented <- ggplot(mean.scented) +
+  aes(x=Date,
+      y=Rating_Mu
+      ) +
+  scale_x_date(date_breaks = "6 months", 
+               date_labels = "%m-%y"
+               ) +
+  scale_y_continuous(name = "Average Daily Rating") +
+  geom_jitter(width = .02, 
```

Comment:
I remember being told not to use "jitter" for this assignment, but this still addresses overplotting